src/main/java/winstone/Launcher.java

```diff
@@ -208,6 +214,22 @@ public Launcher(Map args) throws IOException {
         Runtime.getRuntime().addShutdownHook(new ShutdownHook(this));
     }
 
+    private void writePortToFileIfNeeded() throws IOException {
+        String portFileName = System.getProperty(WINSTONE_PORT_FILE_NAME_PROPERTY);
```

Comment:
I would prefer a command line option (`Option` class) but yeah not sure this will be used a lot :) )